shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -980,12 +970,6 @@ - (void)deregisterNotifications {
 }
 
 - (void)dealloc {
-  // It will be destroyed and invalidate its weak pointers
-  // before any other members are destroyed.
-  _weakFactory.reset();
-
-  // TODO(cbracken): https://github.com/flutter/flutter/issues/157140
-  // Eliminate method calls in initializers and dealloc.
```

Comment:
This PR is a big tech debt win, but not so big we don't still need to track doing this ;)

Reply:
Oh wait, wrong TODO! Nice catch -- thanks. Yeah trying to trace through what the heck is initialiser code and what isn't is a nightmare.

Reply:
Done.